example/savetodisk.js

```diff
@@ -1,73 +1,75 @@
 
-var fs = require('node-fs'),
-    url = require('url'),
-    wrench = require('wrench'),
-    Crawler = require("simplecrawler").Crawler
+var fs = require("node-fs"),
+	url = require("url"),
+	path = require("path"),
+	Crawler = require("../").Crawler;
 
 /**
  * @param String. Domain to download.
  * @Param Function. Callback when crawl is complete.
  */
 var downloadSite = function (domain, callback) {
+	var myCrawler = new Crawler(domain);
+	myCrawler.interval = 250;
+	myCrawler.maxConcurrency = 5;
 
-  // Where to save downloaded data
-  var outputDirectory = __dirname + '/' + domain
-  var myCrawler = new Crawler(domain)
-  myCrawler.interval = 250
-  myCrawler.maxConcurrency = 5
-
-  myCrawler.on("fetchcomplete",function(queueItem, responseBuffer, response) {
-
-    // Parse url
-    var parsed = url.parse(queueItem.url)
-    
-    // Rename / to index.html
-    if (parsed.pathname === '/')
-      parsed.pathname = '/index.html'
-
-    // Get directory name in order to create any nested dirs
-    var dirname = outputDirectory + parsed.pathname.replace(/\/[^\/]+$/, '')
-    
-    // Path to save file
-    var filepath = outputDirectory + parsed.pathname
-
-    // Check if DIR exists
-    fs.exists(dirname, function (exists) {
-
-      // If DIR exists, write file
-      if (exists)
-        fs.writeFile(filepath, responseBuffer, function () {})
-      
-      // Else, recursively create dir using node-fs, then write file
-      else
-        fs.mkdir(dirname, 0755, true, function (err) {
-          fs.writeFile(filepath, responseBuffer, function () {})
-        })
-        
-    })
-
-    console.log("I just received %s (%d bytes)",queueItem.url,responseBuffer.length)
-    console.log("It was a resource of type %s",response.headers['content-type'])
-
-  })
-
-  // Fire callback
-  myCrawler.on('complete', function () {
-    callback()
-  })
-
-  // Start Crawl
-  myCrawler.start()
-  
-}
+	myCrawler.on("fetchcomplete", function(queueItem, responseBuffer, response) {
+
+		// Parse url
+		var parsed = url.parse(queueItem.url);
+
+		// Rename / to index.html
+		if (parsed.pathname === "/") {
+			parsed.pathname = "/index.html";
+		}
+
+		// Where to save downloaded data
+		var outputDirectory = path.join(__dirname, domain);
+
+		// Get directory name in order to create any nested dirs
+		var dirname = outputDirectory + parsed.pathname.replace(/\/[^\/]+$/, "");
+
+		// Path to save file
+		var filepath = outputDirectory + parsed.pathname;
+
+		// Check if DIR exists
+		fs.exists(dirname, function (exists) {
+
+			// If DIR exists, write file
+			if (exists) {
+				fs.writeFile(filepath, responseBuffer, function () {});
+			}
+
+			// Else, recursively create dir using node-fs, then write file
+			else {
+				fs.mkdir(dirname, 0755, true, function () {
+					fs.writeFile(filepath, responseBuffer, function () {});
+				});
+			}
+
+		});
+
+		console.log("I just received %s (%d bytes)", queueItem.url, responseBuffer.length);
```

Comment:
Please stop spamming me with console is not defined. It is defined. As for the line, it's something I don't care in this PR.